specification/apimanagement/resource-manager/readme.md

```diff
@@ -31,12 +31,64 @@ openapi-type: arm
 tag: package-2021-08
```

Comment:
@solankisamir  Please change the default tag to your new tag otherwise SDK pipeline can't check the tag you added.

Reply:
@msyyc We don't release SDK for preview API-versions. We will be creating a stable api-version after this and then we will update this tag.

Reply:
> @msyyc We don't release SDK for preview API-versions. We will be creating a stable api-version after this and then we will update this tag.

@solankisamir Now that the new preview tag will be released for SDK, please review the breaking. @qiaozha @ArcturusZhang